src/core/server/integration_tests/saved_objects/migrations/group2/multi_node_split.test.ts

```diff
@@ -0,0 +1,292 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
```

Comment:
nit: there's another file `multiple_es_nodes`. Should we specify this one to be `multi_kb_node_split`?

Reply:
Accepted proposal, will rename to to `multiple_kb_nodes`